test/NuGet.Core.Tests/NuGet.Configuration.Test/NuGet.Configuration.Test.csproj

```diff
@@ -29,6 +29,6 @@
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
 
-  <Import Project="$(BuildCommonDirectory)common.targets"/>
+  <Import Project="$(BuildCommonDirectory)common.targets" />
```

Comment:
This file only has whitespaces changes. Please revert it back to the dev branch state.